site/app/libraries/FileUtils.php

```diff
@@ -174,6 +174,24 @@ public static function emptyDir($dir) {
         }
     }
 
+    /**
+     * checks to see if directory is empty, if so return true
+     *
+     * @param string $dir
+     */
+    public static function isEmptyDir($dir) {
```

Comment:
Please add native PHP type declarations.

```suggestion
     */
    public static function isEmptyDir(string $dir): bool {
```